AsaApi/Core/Private/Tools/Requests.cpp

```diff
@@ -155,7 +155,7 @@ namespace API
 				}
 				catch (const Poco::Exception& exc)
 				{
-					Log::GetLog()->error(exc.displayText());
```

Comment:
Removing this will just hide the errors that are happening if they are not specifically handled.

Reply:
for handling the errors im redirecting it to response or Results so the user/plugin creator can have more control of it

Reply:
Having both I believe would be acceptable.
```
Log::GetLog()->error(exc.displayText());
Result = exc.displayText();
```